1.0/docs/devguide/local-dom.md

```diff
@@ -226,9 +226,9 @@ You can also locate nodes in the local DOM using `querySelector`, `querySelector
 
     var cancelButton = Polymer.dom(this.root).querySelector('#cancelButton');
```

Comment:
I believe it only returns one:

https://github.com/Polymer/polymer/blob/master/src/standard/utils.html#L28